bootstrap/terraform/main.tf

```diff
@@ -127,29 +139,33 @@ module "eks" {
 
 module "eks_blueprints_addons" {
   source  = "aws-ia/eks-blueprints-addons/aws"
-  version = "1.8.0"
+  version = "~> 1.16"
 
   cluster_name      = module.eks.cluster_name
   cluster_endpoint  = module.eks.cluster_endpoint
   cluster_version   = module.eks.cluster_version
   oidc_provider_arn = module.eks.oidc_provider_arn
-  enable_argocd     = true
+
+  enable_argocd = true
   argocd = {
     namespace     = "argocd"
-    chart_version = "6.3.1" # ArgoCD v2.10.1
+    chart_version = "6.11.1" # ArgoCD v2.11.2
+    timeout       = "600"
     values = [
-      templatefile("${path.module}/argocd-values.yaml", {
+      templatefile("${path.module}/values/control-plane-eks-argocd-stack.yaml", {
```

Comment:
It is nice to organize under values/ folder but why prefix with `control-plane-eks-`?

Reply:
there is no specific reason.. I thought it would be clearer, but I can remove it

Reply:
removed